src/python/txtai/cloud/storage.py

```diff
@@ -2,6 +2,7 @@
 Object storage module
 """
 
+import logging
```

Comment:
This statement has to be removed or the unit/integration tests will fail as it's an unused import.